applications/StructuralMechanicsApplication/custom_processes/set_moving_load_process.cpp

```diff
@@ -0,0 +1,319 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "set_moving_load_process.h"
+#include "utilities/interval_utility.h"
+#include "structural_mechanics_application_variables.h"
+
+
+namespace Kratos
+{
+
```

Comment:
These functions are declared in the entire Kratos namespaces. These must be `private` methods of your process.